tensorrt_llm/commands/bench.py

```diff
@@ -37,18 +38,25 @@
               type=click.Choice(severity_map.keys()),
               default='info',
               help="The logging level.")
+@click.option("--revision",
+              type=str,
+              default=None,
+              help="The revision to use for the HuggingFace model "
+              "(branch name, tag name, or commit id).")
 @click.pass_context
 def main(
     ctx,
     model: str,
     model_path: Path,
     workspace: Path,
     log_level: str,
+    revision: Optional[str],
 ) -> None:
     logger.set_level(log_level)
     ctx.obj = BenchmarkEnvironment(model=model,
                                    checkpoint_path=model_path,
-                                   workspace=workspace)
+                                   workspace=workspace,
+                                   revision=revision)
```

Comment:
I don't think we automatically report out the `BenchEnvironment`, I think it would be a good idea to add this to the output since it's useful info when comparing results.

Reply:
Makes sense, implemented in b1f77f24430fa94a4783150a982b30ef508e193f